src/views/shared/base.njk

```diff
@@ -0,0 +1,127 @@
+{% extends "govuk/template.njk" %}
+{% from "govuk/components/cookie-banner/macro.njk" import govukCookieBanner %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% block head %}
+
+    <!--[if !IE 8]><!-->
+    <link href="/public/stylesheets/application.css" rel="stylesheet">
```

Comment:
should this not be application.scss

Reply:
The scss files from govuk-frontend and our own assets/scss/application.scss are pre-processed into a single minified css file - browsers can't parse scss currently